site/src/pages/404.js

```diff
@@ -0,0 +1,28 @@
+import * as React from "react"
+import { graphql } from "gatsby"
+
+
+const NotFoundPage = ({ data, location }) => {
+  const siteTitle = data.site.siteMetadata.title
```

Comment:
Unused variable...

Reply:
Oh yes. Actually, I just used a starter template of Gatsby to migrate. So, I overlooked at it.